Resources/Locales/ru/openspades.po

```diff
@@ -1,16 +1,16 @@
 # OpenSpades translation
-# Copyright (C) 2016 yvt
+# Copyright (C) 2017 yvt
 # This file is distributed under the same license as the OpenSpades package.
-# yvt <i@yvt.jp>, 2016.
+# yvt <i@yvt.jp>, 2017.
 #
 msgid ""
 msgstr ""
 "Project-Id-Version: OpenSpades\n"
 "Report-Msgid-Bugs-To: i at yvt.jp\n"
-"POT-Creation-Date: 2016-12-25 23:42-0200\n"
-"PO-Revision-Date: 2016-12-25 23:42-0200\n"
-"Last-Translator: ArionWT <arionwhiteac@gmail.com>\n"
-"Language-Team: \n"
+"POT-Creation-Date: 2016-12-25 23:47-0200\n"
+"PO-Revision-Date: 2017-01-03 11:59+0300\n"
+"Last-Translator: ArionWT\n"
+"Language-Team: ArionWT, noway421, feikname, CenturionFox, VelocityRa, DarkDracoon\n"
```

Comment:
[`Language-Team` should be the English name of the language](https://www.gnu.org/software/gettext/manual/html_node/Header-Entry.html).